### PR TITLE
fix bug: @show for simulation

### LINKED ISSUE #168: bug: simulation display
https://github.com/CliMA/ClimaAtmos.jl/blob/main/src/Simulations/simulation.jl#L148

`varnames` is no longer available in model struct and causes error when displaying the simulation struct.

### PR DESCRIPTION
**Pull request by jiahe23**
Close #168

### CODE REVIEW

**Comment by jiahe23** [comment]
bors try

File: src/Simulations/simulation.jl [hunk 1]
```diff
@@ -144,8 +144,8 @@ run!(simulation::Simulation, args...; kwargs...) =
     DiffEqBase.solve!(simulation.integrator, args...; kwargs...)
 
 function Base.show(io::IO, s::Simulation)
-    print(io, "Simulation set-up:\n\tmodel type:\t", typeof(s.model).name.name)
-    print(io, "\n\tmodel vars:\t", s.model.varnames, "\n\n")
+    print(io, "Simulation set-up:\n\tmodel type:\t\t", typeof(s.model).name.name)
+    print(io, "\n\tthermodynamic tar:\t", typeof(s.model.thermodynamics), "\n\n")
```
Comment (bischtob) [inline src/Simulations/simulation.jl:148]: Should it be `var` here?

**Review by bischtob — APPROVED** [review]

**Comment by bischtob** [comment]
Format, squash, rebase, bors r+, please :)

**Comment by jiahe23** [comment]
bors try

File: src/Simulations/simulation.jl (hunk 1 above)
Comment (jiahe23) [reply src/Simulations/simulation.jl]: no such field...

**Pull request merged**